lib/plottr_import_export/src/importer/scrivener/importer.js

```
@@ -95,49 +95,53 @@ function ScrivenerImporter(filePath, isNewFile, state, convertRTFToSlate) {
+      }
 
-    if (places && places.length && places[0]['Children'] && places[0]['Children']['BinderItem']) {
-      const placesBinderItem = Array.isArray(places[0]['Children']['BinderItem'])
-        ? places[0]['Children']['BinderItem']
-        : [places[0]['Children']['BinderItem']]
-      generatePlaces(newJson, placesBinderItem, bookId, files, isNewFile)
-    }
+      if (places && places.length && places[0]['Children'] && places[0]['Children']['BinderItem']) {
+        const placesBinderItem = Array.isArray(places[0]['Children']['BinderItem'])
+          ? places[0]['Children']['BinderItem']
+          : [places[0]['Children']['BinderItem']]
+        generatePlaces(newJson, placesBinderItem, bookId, files, isNewFile)
+      }
 
-    if (
-      characters &&
-      characters.length &&
-      characters[0]['Children'] &&
-      characters[0]['Children']['BinderItem']
-    ) {
-      const charactersBinderItem = Array.isArray(characters[0]['Children']['BinderItem'])
-        ? characters[0]['Children']['BinderItem']
-        : [characters[0]['Children']['BinderItem']]
-      generateCharacters(newJson, charactersBinderItem, bookId, files, isNewFile)
-    }
-    return uuidsToIntegerIds({
-      ...newJson,
-      file: {
-        ...newJson.file,
-        lastOpened: new Date(),
-      },
+      if (
+        characters &&
+        characters.length &&
+        characters[0]['Children'] &&
+        characters[0]['Children']['BinderItem']
+      ) {
+        const charactersBinderItem = Array.isArray(characters[0]['Children']['BinderItem'])
+          ? characters[0]['Children']['BinderItem']
+          : [characters[0]['Children']['BinderItem']]
+        generateCharacters(newJson, charactersBinderItem, bookId, files, isNewFile)
+      }
+      return uuidsToIntegerIds({
+        ...newJson,
+        file: {
+          ...newJson.file,
+          lastOpened: new Date(),
+        },
+      })
+    })
+    .catch((error) => {
+      return new Error(error)
```

Please return the error directly instead.

---

Sorry.  I was wrong here.